drivers/scheduler/k8s/k8s.go

```diff
@@ -4076,6 +4294,225 @@ func (k *K8s) GetTokenFromConfigMap(configMapName string) (string, error) {
 	return token, err
 }
 
+func (k *K8s) createCustomResourceObjects(
```

Comment:
Add a comment above every function that you have written in `k8s.go` to tell what the function is all about

Reply:
✅